### PR TITLE
Nodes addon 1

### PR DESCRIPTION
**Pull request by kosta709**

### CODE REVIEW

File: charts/external-nodes/README.md [hunk 1]
```diff
@@ -0,0 +1 @@
+# codefresh runtim - external nodes
```
Comment (sharon-codefresh) [inline charts/external-nodes/README.md:1]: typo - runtime

File: charts/external-nodes/templates/nodes-cm.yaml [hunk 2]
```diff
@@ -0,0 +1,20 @@
+apiVersion: v1
```
Comment (sharon-codefresh) [inline charts/external-nodes/templates/nodes-cm.yaml:1]: rename the filename to node-configmap

File: charts/external-nodes/templates/runtime-environments-cm.yaml [hunk 3]
```diff
@@ -0,0 +1,7 @@
+apiVersion: v1
```
Comment (sharon-codefresh) [inline charts/external-nodes/templates/runtime-environments-cm.yaml:1]: rename to runtime-environments-configmap.yaml

File: charts/external-nodes/templates/scripts-cm.yaml [hunk 4]
```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
```
Comment (sharon-codefresh) [inline charts/external-nodes/templates/scripts-cm.yaml:1]: rename filename to scripts-configmap.yaml

File: charts/external-nodes/values.yaml [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+    cluster: codefresh
+    protocol: http
+    #alternateLoggerConf: external-nodes
+    nodes:
+    - address: 
+      # name: node1
+      # port: 2376
+    # - address:
+      # name:
+      # port:
```
Comment (sharon-codefresh) [inline charts/external-nodes/values.yaml:23]: if not used for documentation, please remove

File: stage/codefresh/addons/external-nodes/config.yaml [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+cf-api-token: 
+consul: http://cf-consul:8500
+
+## Set runtime environments
+runtimeEnvironments:
+  - name: default-nodes
+    extends: system/default
+    cluster: codefresh
+    protocol: http
+    #alternateLoggerConf: external-nodes
```
Comment (sharon-codefresh) [inline stage/codefresh/addons/external-nodes/config.yaml:28]: if not used, let's remove

File: stage/codefresh/addons/external-nodes/config.yaml [hunk 7]
```diff
@@ -0,0 +1,40 @@
...
+    nodes:
+    - address: 
+      # name: node1
+      # port: 2376
+    # - address:
+      # name:
+      # port:
```
Comment (sharon-codefresh) [inline stage/codefresh/addons/external-nodes/config.yaml:35]: if not used let's remove

File: stage/codefresh/addons/external-nodes/runtime-environments/example-node-1.yaml [hunk 8]
```diff
@@ -0,0 +1,24 @@
+---
```
Comment (sharon-codefresh) [inline stage/codefresh/addons/external-nodes/runtime-environments/example-node-1.yaml:1]: consider to put it under "examples" directory

File: charts/external-nodes/README.md (hunk 1 above)
Comment (kosta709) [reply charts/external-nodes/README.md:1]: fixed in #15

File: charts/external-nodes/templates/nodes-cm.yaml (hunk 2 above)
Comment (kosta709) [reply charts/external-nodes/templates/nodes-cm.yaml:1]: i think -cm is good enough:
* `kubectl get cm` - so it is formal 
* all our cf-helm charts uses this naming

File: charts/external-nodes/templates/runtime-environments-cm.yaml (hunk 3 above)
Comment (kosta709) [reply charts/external-nodes/templates/runtime-environments-cm.yaml:1]: i think -cm is good enough:

kubectl get cm - so it is formal
all our cf-helm charts uses this naming

File: charts/external-nodes/templates/scripts-cm.yaml (hunk 4 above)
Comment (kosta709) [reply charts/external-nodes/templates/scripts-cm.yaml:1]: i think -cm is good enough:

kubectl get cm - so it is formal
all our cf-helm charts uses this naming

File: charts/external-nodes/values.yaml (hunk 5 above)
Comment (kosta709) [reply charts/external-nodes/values.yaml:23]: it is example of config for understanfing

File: stage/codefresh/addons/external-nodes/config.yaml (hunk 6 above)
Comment (kosta709) [reply stage/codefresh/addons/external-nodes/config.yaml:28]: we should show that this option is available somehow

File: stage/codefresh/addons/external-nodes/config.yaml (hunk 7 above)
Comment (kosta709) [reply stage/codefresh/addons/external-nodes/config.yaml:35]: it is example of config for understanding

File: stage/codefresh/addons/external-nodes/runtime-environments/example-node-1.yaml (hunk 8 above)
Comment (kosta709) [reply stage/codefresh/addons/external-nodes/runtime-environments/example-node-1.yaml:1]: done

Comment (kosta709) [reply stage/codefresh/addons/external-nodes/runtime-environments/example-node-1.yaml:1]: done in #15

**Pull request merged**